packages/netlify-cms-core/src/actions/search.ts

```diff
@@ -178,18 +135,12 @@ export function searchEntries(
           searchTerm,
         );
 
-    return searchPromise.then(
-      (response: SearchResponse) =>
-        dispatch(
-          searchSuccess(
-            searchTerm,
-            allCollections as string[],
-            response.entries,
-            response.pagination,
-          ),
-        ),
-      (error: Error) => dispatch(searchFailure(searchTerm, error)),
-    );
+    try {
+      const response: SearchResponse = await searchPromise;
+      return dispatch(searchSuccess(response.entries, response.pagination));
```

Comment:
can 'response.pagination' be different from passed 'page' param?

Reply:
When not using an integration (which is the default), `response.pagination` is missing from `response`